libs/cardano-ledger-api/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 * Addition of `Cardano.Ledger.Api.Tx.Cert`
 * Removal of `VoterRole` in favor of `Voter`
 * Removal of `cgVoterRolesL` and `cgVoterRoles` as no longer needed.
+* Rename `cgTallyL` to `cgGovL`
+* Rename `ConwayTallyState` to `ConwayGovState`
```

Comment:
Forgot to mention in a previous review:
Could you move these two into a new `## 1.4.0.0` section and update cabal version? `1.3.0.0` has already been released, but the changelogs didn't get updated yet.